tf-modules/archive/s3_bucket_lifecycle_config.tf

```diff
@@ -1,16 +1,16 @@
 data "aws_s3_bucket" "system_bucket" {
-  bucket = var.system_bucket 
+  bucket = var.system_bucket
 }
 
 resource "aws_s3_bucket_lifecycle_configuration" "system_bucket_lifecycle_config" {
   bucket = data.aws_s3_bucket.system_bucket.id
   rule {
-    id     = "expire_temporary_execution_status_files"
+    id     = "${var.prefix}_expire_temporary_execution_status_files"
```

Comment:
This is a fix for a bug introduced in https://github.com/nasa/cumulus/pull/3123/files